docs/notifications.md

```diff
@@ -0,0 +1,96 @@
+# Notifications
+
+### Notification Types
+
+#### Package Releases
+
+Private Packagist will notify you every time a new or modified version is discovered in one of the selected packages, e.g. anytime a tag, branch or commit is created or modified in a VCS repository.
+You can specify about which private packages you want to get notified and whether you also want to receive notifications for mirrored packages as well.
+You can filter for releases by minimum stability: "dev / any" will match any release including commits to a branch whereas "stable" will only match releases considered [stable releases by Composer](https://getcomposer.org/doc/articles/versions.md#stability-constraints).
+
+#### Security Alerts
+
+[Security Monitoring](./security-monitoring.md) allows you to select notification channels to receive a notification every time security issues are found in dependencies of one of your projects.
+
+#### Security Summaries
+
+In addition to immediate security alerts your notification channels can also receive either weekly or monthly summaries listing all open security issues in dependencies of your organization's projects.
+
+### Configuring Notifications
+Every user receives security notifications by email for all projects they have access to by default.
+Users can unsubscribe either from individual projects or from all security notifications if they do not wish to receive email notifications.
+
+Notification channels allow you to receive notifications via other means than email to user accounts. The following types of notification channels are available:
+- **Email**: Sends notifications to a list of email addresses
+- **Slack Webhook**: Sends notifications to your configured Slack channel
+- **Microsoft Teams Webhook**: Sends notifications to your configured Microsoft Teams channel
+- **Webhook**: Sends an HTTP POST request to a defined URL optionally signed with a user supplied secret.
+
+Notification channels can be added on your organization’s settings page under *Notification Channels -> Add Notification Channel*.
+
+### Receiving Webhook Notifications
+
+Webhook notifications are sent as HTTP POST requests to the endpoint configured with the notification event data send as payload.
+Client, server, and network errors will automatically be retried up to five times.
```

Comment:
4XX errors can be caused by various different reasons and might because by an invalid setup e.g. 404 because wrong URL posted or 403/401 authentication missing/changed/not setup.

Reply:
@glaubinix then let's not call them client errors, http errors?